UPGRADE.md

```diff
@@ -0,0 +1,116 @@
+# Upgrade Guide
+
+In addition to the README's [Harness Upgrade Instructions], please note these specific version upgrade instructions.
+
+## Upgrading from 0.12.x to 1.0.x
+
+### Removal of docker-sync
+
+If you use docker-sync on macOS, you can convert to mutagen with the following commands:
+```bash
+docker-sync stop
+docker rm "<workspace-project-name>-sync"
```

Comment:
I thought we had mutagen reusing this for the switch?

Reply:
It reuses the volume not the container